cmd/slinky/main.go

```diff
@@ -343,3 +357,56 @@ func overwriteMarketMapEndpoint(cfg config.OracleConfig, overwrite string) (conf
 
 	return cfg, fmt.Errorf("no market-map provider found in config")
 }
+
+// filterToOnlyCMCMarkets is a helper function that filters out all markets that are not from CoinMarketCap. It
+// mutates the marketmap to only include CoinMarketCap markets.
+func filterToOnlyCMCMarkets(marketmap mmtypes.MarketMap) mmtypes.MarketMap {
```

Comment:
just to note that this is for the dydx chain so all of these IDs are just the ID of the BASE asset. So they will always be prices in BASE/USD